compiler/aarch64/runtime/ARM64arrayCopy.spp

```diff
@@ -24,10 +24,82 @@
 	.global	__arrayCopy
 	.global	__forwardArrayCopy
 	.global	__backwardArrayCopy
+	.global	__fwHalfWordArrayCopy
+	.global	__fwWordArrayCopy
+	.global	__fwDoubleWordArrayCopy
+	.global	__fwQuadWordArrayCopy
+	.global	__bwHalfWordArrayCopy
+	.global	__bwWordArrayCopy
+	.global	__bwDoubleWordArrayCopy
+	.global	__bwQuadWordArrayCopy
 
 	.text
 	.align	2
 
+// These are entry points for halfword, word and doubleword alignment of arrays to be copied
+// in forward direction to avoid multiple checks in generic forward copy entry point,
+// when the alignment is known.
+//
+// in:    x0 - length in bytes
+//        x1 - src addr
+//        x2 - dst addr
+
+__fwHalfWordArrayCopy:
+	cbz	x0, finished			// return if no bytes to copy
+	cmp x1, x2
```

Comment:
Please use a TAB character between `cmp` and `x1, x2`.
The same comment for many other lines.